instance/view.go

```diff
@@ -92,7 +92,7 @@ func (vo *viewOperation) Run(serviceInstanceAdminURL string, accessToken string)
 }
 
 func (vo *viewOperation) IsLifecycleOperation() bool {
-	return false
+	return true
```

Comment:
This makes me think if the name `IsLifeCycleOperation` is accurate. Initially only `stop`, `start`, `restage` and `restart` were considered lifecycle operations but `view` doesn't seem to fit well...Not sure if we can find a better name (`useServiceBrokerUriInV3`?)

Reply:
`IsServiceBrokerEndpoint` perhaps?

Reply:
Could be, but it only applies in the context of version 3. But at least it's better than the current name after your change, so happy to use that one :)

Reply:
The statement is true none the less :-)